apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js

```diff
@@ -81,6 +81,40 @@ function calculateCompletedAt (resolvedData, existedStatus, resolvedStatus) {
     }
 }
 
+function calculateIsCompletedAfterDeadline (resolvedData, existingItem) {
+    const newItem = { ...existingItem, ...resolvedData }
+    const completedAt = get(newItem, 'completedAt')
+    const deadline = get(newItem, 'deadline')
+    const statusUpdatedAt = get(newItem, 'statusUpdatedAt')
+
+    const prevStatusId = get(existingItem, 'status')
+    const nextStatusId = get(newItem, 'status')
+
+    let stopPoint
+
+    if (nextStatusId === STATUS_IDS.DECLINED) {
+        if ((prevStatusId === STATUS_IDS.COMPLETED || prevStatusId === STATUS_IDS.CLOSED || nextStatusId === STATUS_IDS.DECLINED) && completedAt) {
```

Comment:
here you check `nextStatusId === STATUS_IDS.DECLINED` twice

Reply:
fixed